internal/pkg/manifest/env.go

```diff
@@ -537,20 +547,13 @@ func (cfg PublicHTTPConfig) IsEmpty() bool {
 }
 
 type privateHTTPConfig struct {
-	InternalALBSubnets   []string             `yaml:"subnets,omitempty"`
-	Certificates         []string             `yaml:"certificates,omitempty"`
-	SecurityGroupsConfig securityGroupsConfig `yaml:"security_groups,omitempty"`
+	InternalALBSubnets   []string                          `yaml:"subnets,omitempty"`
+	Certificates         []string                          `yaml:"certificates,omitempty"`
+	SecurityGroupsConfig DeprecatedALBSecurityGroupsConfig `yaml:"security_groups,omitempty"`
```

Comment:
nit:
```suggestion
	SecurityGroupsConfig DeprecatedALBSecurityGroupsConfig `yaml:"security_groups,omitempty"` // Deprecated.
```
Adding the comment (like you do with the `publicHTTPConfig` struct) helps the IDE to show the field differently!